reinforcement_learning/rlclientlib/logger/event_batcher.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "event_queue.h"
+#include "ranking_event.h"
+#include "utility/data_buffer.h"
+
+namespace reinforcement_learning {
+  namespace utility {
+    class event_batcher {
```

Comment:
I'm not sure I understand the difference between event_batcher and async_batcher. Could you explain?

Reply:
event_batcher is to create the eventhub messages (which is an flatbuffer object, that logically contains a batch of events).

async_batcher was just previous logic, which process the eventqueue and send messages.

Reply:
Given this changes how they interact can you please rename to make this clearer?